Update acm_deprecate_remove.adoc (#7944)
open-cluster-management/rhacm-docs: release_notes/acm_deprecate_remove.adoc

| Enable the _Fleet view_ switch to view the new default _Overview_ page.
| The previous layout of the {acm-short} _Overview_ page is redesigned.

| Installer
| `ingress.sslCiphers` field in `operator.open-cluster-management.io_multiclusterhubs_crd.yaml`
| 2.9
| None

> | See link:../install/adv_config_install.adoc[Advanced Configuration] for configuring install. If you uppgrade your {acm} version and originally had a `MultiClusterHub` custom resource with the `spec.ingress.sslCiphers` field defined, the field is still recognized, but is deprecated and has no effect.

Reviewer: @ngraham20 is there a different parameter that users should look at? When something is removed, I expect that the user would not see the parameter. It seems like we should remove the statement.
```suggestion
| See link:../install/adv_config_install.adoc[Advanced Configuration] for configuring install. If you uppgrade your {acm} version and originally had a `MultiClusterHub` custom resource with the `spec.ingress.sslCiphers` field defined, the field is still recognized, but is removed.
```
